src/patternfly/components/TreeView/tree-view-node-check.hbs

```diff
@@ -2,8 +2,8 @@
   {{#if tree-view-node-check--attribute}}
     {{{tree-view-node-check--attribute}}}
   {{/if}}>
-  {{#if tree-view-node--HasCheckboxSelected}}
-    <input type="checkbox" checked aria-label="Tree view node check checked" />
+  {{#if tree-view-node--IsChecked}}
+    <input type="checkbox" id="check-{{{tree-view-node--id}}}" checked aria-label="Tree view node check checked" />
```

Comment:
Is the self closing `/` needed in our workspace? I don't think it is since it isn't JSX.

```suggestion
    <input type="checkbox" id="check-{{{tree-view-node--id}}}" checked aria-label="Tree view node check checked">
```